filecopy/operations/models.py

```diff
@@ -116,9 +116,17 @@ def owner(self) -> str:
 
     @property
     def namespace(self) -> str:
-        result = {1: 'core'}.get(self.get('zone'), 'greenroom')
+        result = {1: 'Core'}.get(self.get('zone'), 'Greenroom')
         return result
 
+    @property
+    def zone(self) -> str:
```

Comment:
As I understand here zone suppose to be `int`.
```suggestion
    def zone(self) -> int:
```